### PR TITLE
fix attributeerror in tutorial part10

### PR DESCRIPTION
**Pull request by Rachelxuan11**
**Problem**
Run tutorial Part10 and got the same error as this issue #2758. 
This error is shown as follows:

> <ipython-input-6-c4374ada1b4c> in train_enc()
     17             spdz_params = list()
     18             for remote_index in range(len(compute_node)): # for each client
---> 19                 spdz_params.append(params[remote_index][param_i].copy().fix_precision().share(bob, alice, crypto_provider=james).get()) # AttributeError: 'numpy.ndarray' object has no attribute 'wrap', if .get() is used here. This could be avoided by model.get() first, and then .share() the copy to a list of clients. This obeys the actually protocol of encrypted model averaging, but fixes this bug.

**Solution**
user `model.copy().get().fix_precision().share(...)` instead of `model.copy().fix_precision().share(...).get()` could avoid this error

### CODE REVIEW

File: examples/tutorials/Part 10 - Federated Learning with Secure Aggregation.ipynb [hunk 1]
```diff
@@ -197,7 +197,7 @@
     "        if batch_idx % args.log_interval == 0:\n",
     "            loss = loss.get()\n",
     "            print('Train Epoch: {} [{}/{} ({:.0f}%)]\\tLoss: {:.6f}'.format(\n",
-    "                epoch, batch_idx * data.shape[0], len(train_loader),\n",
+    "                epoch, batch_idx * data.shape[0], len(train_loader) * data.shape[0],\n",
     "                       100. * batch_idx / len(train_loader), loss.item()))\n",
```
Comment (Rachelxuan11) [inline examples/tutorials/Part 10 - Federated Learning with Secure Aggregation.ipynb]: a small change

File: examples/tutorials/Part 10 - Federated Learning with Secure Aggregation.ipynb [hunk 2]
```diff
@@ -221,7 +221,6 @@
     "    for data, target in test_loader:\n",
     "        output = model(data)\n",
     "        test_loss += F.mse_loss(output.view(-1), target, reduction='sum').item() # sum up batch loss\n",
-    "        pred = output.data.max(1, keepdim=True)[1] # get the index of the max log-probability\n",
```
Comment (Rachelxuan11) [inline examples/tutorials/Part 10 - Federated Learning with Secure Aggregation.ipynb]: redundant for this example regression task

File: examples/tutorials/Part 10 - Federated Learning with Secure Aggregation.ipynb [hunk 3]
```diff
@@ -458,7 +457,7 @@
     "        for param_i in range(len(params[0])):\n",
     "            spdz_params = list()\n",
     "            for remote_index in range(len(compute_nodes)):\n",
-    "                spdz_params.append(params[remote_index][param_i].copy().fix_precision().share(bob, alice, crypto_provider=james).get())\n",
+    "                spdz_params.append(params[remote_index][param_i].copy().get().fix_precision().share(bob, alice, crypto_provider=james)\n",
```
Comment (Rachelxuan11) [inline examples/tutorials/Part 10 - Federated Learning with Secure Aggregation.ipynb]: avoid the AttributeError

**Comment by gmuraru** [comment]
It seems that there is an error here (it is lalso present on master, but also in this PR). Do you have some time to investigate :)
```
~/contrib/tests/PySyft/syft/generic/frameworks/hook/hook_args.py in <lambda>(i, **kwargs)
    742         if isinstance(r, (list, tuple))  # if the rule is a list or tuple.
    743         # Last if not, rule is probably == 1 so use type to return the right transformation.
--> 744         else lambda i, **kwargs: register_tensor(i, **kwargs)
    745         for a, r in zip(response, rules)  # And do this for all the responses / rules provided
    746     ]

~/contrib/tests/PySyft/syft/generic/frameworks/hook/hook_args.py in register_tensor(tensor, owner, response_ids)
    710             and each id is pop out when needed.
    711     """
--> 712     tensor.owner = owner
    713     try:
    714         tensor.id = response_ids.pop(-1)

AttributeError: 'numpy.ndarray' object has no attribute 'owner'
```

**Pull request closed without merging**